components/IndoorDirections/TypesOfDirections/SameFloorDirections.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Line, G } from 'react-native-svg';
+import { ClassGraph } from '../../../constants/ClassGraph';
+import { dijkstra, getFloorNumber, changeClassName } from '../Dijkstra/DijkstraAlgorithm';
+
+export function SameFloorDirections(props) {
+    const rooms = props.rooms;
+    const graph = ClassGraph();
+    const from = changeClassName(props.from);
```

Comment:
Here you didn't write an if that checks if the string includes a space. hmmm

Reply:
Since we know you are in the same floor, classrooms do not have spaces

Reply:
I'm not sure to understand. Is it a feature of your algorithm, having spaces or you referencing the values from the constants?

Reply:
The only reason you need to check for spaces is if you will be receiving a string that is not a classroom. In _SameFloorDirections_, all you receive are two classrooms

Reply:
okok I understand!